### ISSUE TITLE
super_resolution model building question

### ISSUE THREAD

**Issue by pageedward** [pytorch/examples#163]
class Net(nn.Module):
    def __init__(self, upscale_factor):
        super(Net, self).__init__()

        self.relu = nn.ReLU()
        self.conv1 = nn.Conv2d(1, 64, 5, 1, 2)
        self.conv2 = nn.Conv2d(64, 64, 3, 1, 1)
        self.conv3 = nn.Conv2d(64, 32, 3, 1, 1)
        self.conv4 = nn.Conv2d(32, upscale_factor ** 2, 3, 1, 1)
        self.pixel_shuffle = nn.PixelShuffle(upscale_factor)

how could u get the following information from the paper,
in the self.conv1,  there is padding = 2
layer num l in paper is 3 ,why do u add self.conv2,?
self.conv4 's output_channel is upscale_factor**2

**Comment by Kaixhin** [comment]
> This example illustrates how to use the efficient sub-pixel convolution layer

The code states that this is an example of how to use the layer, not a replica of the model from the associated paper.

The amount of padding is chosen to [make sure the output size is the same as the input size](https://stackoverflow.com/questions/37674306/what-is-the-difference-between-same-and-valid-padding-in-tf-nn-max-pool-of-t).
